server/src/interfaces.ts

```diff
@@ -22,6 +22,9 @@ import {Operator} from '@google-cloud/datastore/build/src/query';
  */
 export interface CustomerPayload {
   gpayId: string;
+  // Phone number in the format of country code appended by national number (as defined
+  // by the International Telecommunication Union (ITU)) and separated by space.
```

Comment:
can you write an example of such phone number format to the comment?

Reply:
+1

Reply:
Actually Karen can we change this to 
```
  // gpayContactNumber format is E.123 international notation
  // Note that the national phone number portion will only contain digits (no spaces)
  // Eg: +91 1234567890
```
It's a description that's partially taken from Spot API docs as well haha. I feel that it's more specific also!

Reply:
@ifhuang @xlai20 Okay!
@hzxcaryn Okay! I put that comment but also added one more line.